packages/Icon/stories/examples/AllIcons.js

```diff
@@ -1,20 +1,34 @@
 import React from "react";
-import Heading from "@paprika/heading";
+import { select, color } from "@storybook/addon-knobs";
 import stylers from "@paprika/stylers";
-
+import tokens from "@paprika/tokens";
+import Heading from "@paprika/heading";
 import * as sc from "./AllIcons.styled";
 
-export default function AllIcons() {
+const iconSizes = {
+  small: stylers.spacer(2),
+  medium: stylers.spacer(3),
+  large: stylers.spacer(4),
+};
+
+export default () => {
+  const iconProps = {
+    size: select("size", iconSizes, stylers.spacer(3)),
+    color: color("color", tokens.textColor.icon),
+  };
+
   const req = require.context("../../src", false, /\.js$/);
```

Comment:
why is this still working? I thought you have to import from `../../lib` since you removed those js files from `src`

Reply:
This script runs beforehand, and will create all those `.js` files in `/src`:
https://github.com/acl-services/paprika/blob/908f89ee9d59397a280264f74da96b308aa61469/packages/Icon/package.json#L17

Reply:
should those be gitignored then?

Reply:
sure!

Reply:
@alexzherdev 
Tokens ignores are in the root... should these also go in the same file? 
https://github.com/acl-services/paprika/blob/master/.gitignore#L36-L40
Or do you think both of  should be in a local `.gitignore`?

Reply:
both have pros and cons... I'd probably put them locally. Make sure that publishing still works. There is an interplay between `.gitignore` and `.npmignore` but AFAICT we should be fine https://docs.npmjs.com/using-npm/developers.html#keeping-files-out-of-your-package